cli/src/cli.rs

```diff
@@ -400,23 +389,37 @@ pub async fn init_command(
     secret_manager: SecretManager,
     init_params: InitParameters,
 ) -> Result<Wallet, Error> {
-    let alias = if get_decision("Do you want to assign an alias to your wallet?")? {
-        Some(get_alias("New wallet alias").await?)
-    } else {
-        None
-    };
+    let mut address = init_params.address.map(|s| Bech32Address::from_str(&s)).transpose()?;
+    if address.is_none() {
+        if get_decision("Do you want to assign an existing address to your new wallet?")? {
+            address.replace(get_address("Set wallet address").await?);
+        }
+    }
+
+    let mut bip_path = init_params.bip_path;
+    if bip_path.is_none() {
+        if get_decision("Do you want to assign a valid address bip path to your new wallet?")? {
```

Comment:
`valid address` sounds useless here or can I set a bip path for an invalid address? :trollface:

Reply:
it was meant to be read as "valid address-bip-path", but I gotcha :smiley:

Reply:
Changed